linkeddatanotifications-subscription-2021.html

```diff
@@ -0,0 +1,485 @@
+              <p>This specification defines the <code>LinkedDataNotificationsSubscription2021</code> type for use with Solid Notifications subscriptions. The URI of the subscription type is <code>http://www.w3.org/ns/solid/notification#LinkedDataNotificationsSubscription2021</code>.</p>
+
+              <p class="issue">The URI and definition of the subscription type is pending the approval of JSON-LD Context and the Notification vocabulary: <a href="https://github.com/solid/vocab/pull/62/">https://github.com/solid/vocab/pull/62/</a>.</p>
+
+              <p>An <code>LinkedDataNotificationsSubscription2021</code> API MUST conform to the <cite><a href="https://solid.github.io/notifications/protocol#discovery">Solid Notifications Protocol</a></cite> [<cite><a class="bibref" href="#bib-solid-notifications">SOLID-NOTIFICATIONS</a></cite>].</p>
+
+              <p>An <code>LinkedDataNotificationsSubscription2021</code> API SHOULD support the <cite><a href="https://solid.github.io/notifications/protocol#notification-features">Solid Notifications Features</a></cite>.</p>
+
+              <p property="skos:definition">The <code>LinkedDataNotificationsSubscription2021</code> type defines the following properties:</p>
+
+              <span rel="skos:hasTopConcept"><span resource="#linkeddatanotificationssubscription2021-inbox"></span><span resource="#linkeddatanotificationssubscription2021-webid"></span></span>
+
+              <dl>
+                <dt about="#linkeddatanotificationssubscription2021-inbox" property="skos:prefLabel" typeof="skos:Concept"><dfn id="linkeddatanotificationssubscription2021-inbox">inbox</dfn></dt>
+                <dd about="#linkeddatanotificationssubscription2021-inbox" property="skos:definition"><span about="" id="linkeddatanotificationssubscription2021-inbox-https" rel="spec:requirement" resource="#linkeddatanotificationssubscription2021-inbox-https"><span property="spec:statement">The <code>inbox</code> property is used in the body of the subscription request. The value of this property <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> be a URI, using the <code>https</code> scheme.</span></span> <span about="" id="linkeddatanotificationssubscription2021-inbox-ldn-receiver" rel="spec:requirement" resource="#linkeddatanotificationssubscription2021-inbox-ldn-receiver"><span property="spec:statement">The inbox URI <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> conform to the <a href="https://www.w3.org/TR/ldn/#receiver" rel="cito:citesAsAuthority">receiver</a> portion of the <a href="https://www.w3.org/TR/ldn/">Linked Data Notifications</a> [<cite><a class="bibref" href="#bib-ldn">LDN</a></cite>] specification.</span></span></dd>
+                <dt about="#linkeddatanotificationssubscription2021-webid" property="skos:prefLabel" typeof="skos:Concept"><dfn id="linkeddatanotificationssubscription2021-webid">webid</dfn></dt>
```

Comment:
I like this approach. I will look at PR-ing similar one for WebHooks so that Solid-OIDC can be used by publishers to deliver notifications.